ci/draft-release.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
```

Comment:
Did you write all these new shell scripts from scratch?

Do you expect these scripts to run during ci?  If not perhaps they should just live under `scripts/` or at the top level like the rest of the current `.sh` scripts?

Reply:
Yes, I wrote them as I fought through issues with CI/Git/GitHub while trying to publish the wasi-sdk-20+threads pre-release (it was a fight: not every CI run has the right tag applied, e.g.). I had to do things enough times that this work already paid for itself, but in the back of my head it seemed like a good idea to record how one might publish new wasi-sdk releases. I was hoping at some point to integrate these in CI with a GitHub workflow trigger but that could be done later. I don't mind moving them to `scripts` or the top-level directory... tell me which is best and I'll move them. (Maybe we move all the `*.sh` files to `scripts`?).

Reply:
I talked to @sunfishcode a bit more about this kind of PR and I think I'm going to go ahead and merge soon; @sbc100, any strong preference on where to put these scripts?

Reply:
Not really no.